### PR TITLE
Feature/useMediator

### PR DESCRIPTION
**Pull request by SolovyovDmitry**
Use mediator pattern in find offices endpoint

### CODE REVIEW

**Comment by VLiamin** [comment]
May be better to replace all classes to folder Commands/Office/**Find**. Because there are three endpoints in OfficeController.(find, edit and create)

**Comment by VLiamin** [comment]
Also may be create DataLayer like here(https://github.com/LT-Students/DigitalOffice-SurveyService/tree/small-refactoring/src)?

**Comment by SolovyovDmitry** [comment]
> Also may be create DataLayer like here(https://github.com/LT-Students/DigitalOffice-SurveyService/tree/small-refactoring/src)?

I'd put this decision away till there would be decision to rework whole service

File: src/OfficeService.Business/Commands/Office/Find/FindOfficesHandler.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+      if (filter.IsActive.HasValue)
+      {
+        offices = offices.Where(x => x.IsActive == filter.IsActive);
+      }
+
+      if (!string.IsNullOrWhiteSpace(filter.NameIncludeSubstring))
+      {
+        offices = offices.Where(x =>
+          x.Name.Contains(filter.NameIncludeSubstring));
```
Comment (MariyaSin) [inline src/OfficeService.Business/Commands/Office/Find/FindOfficesHandler.cs:43]: ignor case

**Review by MariyaSin — APPROVED** [review]

**Review by MariyaSin — COMMENTED** [review]
Update specification in postman

**Review by vorobevamv — APPROVED** [review]

**Pull request merged**